app/views/content/non-uk-teachers/_why-teach.html.erb

```diff
@@ -1,10 +1,11 @@
 <section class="why-teach" aria-label="Why teach?">
   <div class="why-teach__container">
     <div class="why-teach__content">
-      <h2 class="heading-l">
-        <div class="caption-l caption--bold caption--purple">Why teach in England?</div>
-        Join a world class education system
-      </h2>
+      <%= render Content::HeadingComponent.new(
+        heading: "Join a world class education system",
+        tag: :h2,
+        caption: "Why teach in England?",
```

Comment:
@gazcarr This caption (Why teach in England) is quite a bit smaller now - are you happy with this? I'm fine either way

https://review-get-into-teaching-app-2975.london.cloudapps.digital/non-uk-teachers